### DISCUSSION TITLE
problems with icons @expo/vector-icons@14.0.2

### DISCUSSION THREAD

**Discussion by MouadTs** [expo/expo#28791] [Archived: Questions]
after updating to SDK 51 and also updating expo vector as well the  icons in my react native app don't seem to load and i get some error related to Module 6 or 17 that are missing from the aset registry .i've tried deleting expo vector manually and reinstall it but nothing seems to work, this is the error : Possible unhandled promise rejection (id: 5):
Error: Module "2" is missing from the asset registry
Call Stack
fromModule
C:\Users\Lenovo\Desktop|Stays...odules\expo-asset\build Asset.js
getAssetForSource
C:\Users\Lenovo\Desktop|Stays...es\expo-font\build\FontLoader.js
‹anonymous>

**Comment by lukmccall** [comment]
Please check if you have a duplicate of `@react-native/asset-registry`. If you are using `yarn`, you can run `yarn why @react-native/asset-registry` to see which versions are installed. This type of error indicates that you may have multiple versions installed in your project, which is not correct.
